flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -505,10 +511,10 @@ void createTemporarySystemFunction(
      * (implicitly or explicitly) identified by a catalog and database.
      *
      * @param path The path under which the function will be registered. See also the {@link
-     *     TableEnvironment} class description for the format of the path.
+     *         TableEnvironment} class description for the format of the path.
      * @param functionClass The function class containing the implementation.
      * @param ignoreIfExists If a function exists under the given path and this flag is set, no
-     *     operation is executed. An exception is thrown otherwise.
+     *         operation is executed. An exception is thrown otherwise.
```

Comment:
a little confusion : The change was caused by `mvn spotless:apply` command ?

Reply:
Yes, it was caused by auto style check.